themes/ropensci/static/css/style-new.css

```diff
@@ -52,106 +52,83 @@ q:before, q:after {
     width: 1200px;
 }.row{
     box-sizing:border-box;
-    display:-webkit-box;
     display:flex;
-    -webkit-box-flex:0;
-            flex:0 1 auto;
-    -webkit-box-orient:horizontal;
-    -webkit-box-direction:normal;
-            flex-direction:row;
+    flex:0 1 auto;
+    flex-direction:row;
     flex-wrap:wrap;
 }.middle-container {
     margin: 0 auto;
-    display:-webkit-box;
     display:flex;
-    -webkit-box-orient: horizontal;
-    -webkit-box-direction: normal;
-            flex-direction: row;
+    flex-direction: row;
     flex-wrap: wrap;
-    -webkit-box-flex: 0;
-            flex: 0 1 auto;
-    -webkit-box-pack: center;
-            justify-content: center;
+    flex: 0 1 auto;
+    justify-content: center;
     align-content: stretch;
-    -webkit-box-align: center;
-            align-items: center;
+    align-items: center;
     box-sizing: border-box;
     height: 90vh;
 }.col {
-    -webkit-box-flex: 1;
     flex-grow: 1;
     flex-basis: 0;
     max-width: 100%;
 }.col-1{
   box-sizing: border-box;
-  -webkit-box-flex: 0;
-          flex: 0 0 auto;
+  flex: 0 0 auto;
   padding-left: .25%;
   padding-right: .25%;
 }.col-2{
   box-sizing: border-box;
-  -webkit-box-flex: 0;
```

Comment:
not sure why this was changed, maybe a css/npm update?

Reply:
It's possible but Im not sure.